.travis.yml

```diff
@@ -10,23 +10,46 @@ python:
   - "3.7-dev"
   - "pypy"
   - "pypy3"
+
 cache:
   apt: true
   pip: false
-env: PYCURL_SSL_LIBRARY=gnutls
+  directories:
+    - stripe-mock
+
+env:
+  global:
+    - PYCURL_SSL_LIBRARY=gnutls
+    - STRIPE_MOCK_VERSOON=0.1.12
```

Comment:
I think there's a typo in "version" here. Also, maybe bump this to 0.1.17 because I fixed a couple bugs since :)